backend/api/filter.proto

```diff
@@ -115,4 +119,6 @@ message Filter {
   repeated Predicate predicates = 1;
 }
 
-
+service DummyFilterService {
```

Comment:
do we need this?

Reply:
Yes, otherwise we don't get the swagger definitions being generated for Filter message. Added clarifying comments